backend/tracim_backend/applications/content_html_document/controller.py

```diff
@@ -285,6 +285,70 @@ def set_html_document_status(self, context, request: TracimRequest, hapic_data=N
             api.save(content)
         return
 
+    @hapic.with_api_doc(tags=[SWAGGER_TAG__CONTENT_HTML_DOCUMENT_ENDPOINTS])
+    @check_right(is_reader)
+    @check_right(is_html_document_content)
+    @hapic.input_query(FileQuerySchema())
+    @hapic.input_path(FilePathSchema())
+    @hapic.output_file([])
+    def preview_pdf_full(self, context, request: TracimRequest, hapic_data=None):
+        """
+        Obtain a full pdf preview (all page) of last revision of content.
+        Good pratice for filename is filename is `{label}.pdf`.
+        Default filename value is 'raw' (without file extension) or nothing.
+        """
+        app_config = request.registry.settings["CFG"]  # type: CFG
+        api = ContentApi(
+            show_archived=True,
+            show_deleted=True,
+            current_user=request.current_user,
+            session=request.dbsession,
+            config=app_config,
+        )
+        content = api.get_one(hapic_data.path.content_id, content_type=content_type_list.Any_SLUG)
+        default_filename = "{label}.pdf".format(label=content.label)
+        result = api.get_full_pdf_preview_from_html_raw_content(
+            revision=content.revision,
+            filename=hapic_data.path.filename,
+            default_filename=default_filename,
+            force_download=hapic_data.query.force_download,
+            additional_metadata={},
```

Comment:
As `additional_metadata` seems unused I would rather remove it.

Reply:
done.